staging/src/k8s.io/client-go/util/conntrack/conntrack.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package conntrack implements a connection dialer that tracks and can close
```

Comment:
Please document: Why would I want to do this? What problem does this solve?

Consider putting the package comment in a doc.go file. (not mandatory since this seems to be a one file package)

Reply:
Added a bit more on purpose of the package.